.editorconfig

```diff
@@ -5,7 +5,7 @@
 # top-most EditorConfig file
 root = true
 
-[*.hs]
+[*.{hs, ts, tsx, scss, json, js, svg}]
```

Comment:
How to apply this style guide to VSCode? Will it automatically be applied? What about IntelliJ?

I'll test with VSCode now.

Reply:
VSCode works. We need to test Emacs @mkaito and IntelliJ @sashasashasasha151

Reply:
This .editorconfig is most primary way to format. Even Idea overrides its rules of formatting with this file. So I expect all modern editors take into consideration this file as well